ci-operator/config/openshift-pipelines/performance/tools/manage-perf-crons.py

```
@@ -0,0 +1,731 @@
+
+    texts = {}
+    for _, rep_name, new_cron in ops:
+        if not rep_name or rep_name not in bmap:
+            continue
+        s, e = bmap[rep_name]
+        text = "".join(lines[s:e])
+        text = re.sub(r"  cron: .+\n", f"  cron: {new_cron}\n", text)
+        texts[(rep_name, new_cron)] = text
+
+    if need_clone:
+        for pos_name, rep_name, new_cron in ops:
```

Ruff (0.15.21)

[warning] Loop control variable `pos_name` not used within loop body

(B007)